.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+keys.json
```

Comment:
if you are excluding node_modules then there is no need to include the `package-lock.json` file.